src/test/kotlin/app/batch/ObjectSizeFilterTest.kt

```diff
@@ -0,0 +1,67 @@
+package app.batch
+
+import app.domain.InputStreamPair
+import ch.qos.logback.classic.spi.ILoggingEvent
+import ch.qos.logback.core.Appender
+import com.nhaarman.mockitokotlin2.argumentCaptor
+import com.nhaarman.mockitokotlin2.mock
+import com.nhaarman.mockitokotlin2.times
+import com.nhaarman.mockitokotlin2.verify
+import org.junit.Assert.*
+import org.junit.Test
+import org.slf4j.LoggerFactory
+import java.io.ByteArrayInputStream
+
+class ObjectSizeFilterTest {
+
+    @Test
+    fun test_WhenFileUnderTheMaximumSize_ThenItIsAccepted() {
+        val objectSizeFilter = ObjectSizeFilter(100)
+        val dataStream = ByteArrayInputStream("DATA".toByteArray())
+        val metadataStream = ByteArrayInputStream("METADATA".toByteArray())
+        val inputStreamPair = InputStreamPair(dataStream, metadataStream,"S3_KEY", 50)
```

Comment:
I'd make this 99 and the one below 101, put them at the boundaries unless there's a reason not to explicitly.